doc-first-package.md

```diff
@@ -0,0 +1,56 @@
+# My first package walk-through
+You're missing a good analysis tools in this package repository?
+See below on how you can create the package via a GitHub issue or manually with the support of a package template script.
+
+## Automatic pull request via new issue
+* go to Issues - New Issue
+* select the appropriate package type
+  * for a "METAPACKAGE" select the respective option, all other packages select "NEW PACKAGE"
+* fill out the required fields
+* submit your new issue
+* a maintainer will review your submission and label it for automatic package creation
+  * the automated processing includes linting and testing
+* if changes are required, you can
+  * download the created branch and modify it locally
+  * select the branch and edit via the GitHub web UI 
+
+## Manual package creation
+* find the appropriate package type
+  * see supported options via `$ python scripts/utils/create_package_template.py --type`
+* see required arguments via `$ python scripts/utils/create_package_template.py --type <package_type>`
+* provide required arguments, for example for:
+  * type `METAPACKAGE`
+  ```
+  $ python scripts/utils/create_package_template.py \
+    --type METAPACKAGE \
+    --pkg_name dnspyex \
+    --tool_name dnSpyEx \
+    --version 6.2.0 \
+    --category dotNet \
+    --authors '0xd4d, ElektroKill' \
+    --description 'dnSpyEx is ...' \
+    --dependency 'dnSpyEx' \
+    --shim_path '%PROGRAMDATA%\chocolatey\bin\dnSpy.exe'
+  ``` 
+  * type `ZIP_EXE`
+  ```
+  $ python scripts/utils/create_package_template.py \
+    --type ZIP_EXE \
+    --pkg_name floss \
+    --tool_name flare-floss \
+    --version 2.1.0 \
+    --authors Mandiant \
+    --description 'FLOSS ...' \
+    --category Utilities \
+    --target_url https://github.com/mandiant/flare-floss/releases/download/v2.1.0/floss-v2.1.0-windows.zip \
+    --target_hash 925df10403b45e29914e44ac50d92d762b2b2499c11cdd1801888aac95b53eb7
+  ```
+* ensure scripts and package meet code and structure requirements
```

Comment:
I would mention here also the documentation: https://github.com/mandiant/VM-Packages/wiki/Contributing

Reply:
added